pom.xml

```diff
@@ -132,6 +132,14 @@
             <scope>compile</scope>
         </dependency>
 
+        <!-- To avoid javadoc warnings. -->
```

Comment:
Do we use findbugs in any other SDK? Has George approved this?

Reply:
This kind of change needs to go through Mike B and how does it avoid javadoc warnings?

Reply:
I reverted this change. We seems one warning is not worth the trouble.